src/Microsoft.TemplateEngine.TemplateLocalizer.Core/ExportResult.cs

```diff
@@ -7,12 +7,19 @@ namespace Microsoft.TemplateEngine.TemplateLocalizer.Core
 {
     public sealed class ExportResult
     {
-        public string? TemplateJsonPath { get; set; }
+        public ExportResult(string? templateJsonPath, string? errorMessage, Exception? innerException)
```

Comment:
minor:
consider overloading to avoid optional params in .ctors...

`public ExportResult(string? templateJsonPath) :this (templateJsonPath, null){ }`

`public ExportResult(string? templateJsonPath, string? errorMessage) :this(templateJsonPath, templateJsonPath, null) { }`

relevant discussion: https://softwareengineering.stackexchange.com/questions/368414/optional-parameters-or-overloaded-constructors